.github/workflows/python-publish.yml

```diff
@@ -16,21 +10,21 @@ jobs:
   deploy:
 
     runs-on: ubuntu-latest
+    permissions:
+      # IMPORTANT: this permission is mandatory for trusted publishing
+      id-token: write
 
     steps:
     - uses: actions/checkout@v4
     - name: Set up Python
       uses: actions/setup-python@v5
       with:
-        python-version: 3.7
+        python-version: 3.9
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install setuptools wheel twine
-    - name: Build and publish
-      env:
-        TWINE_USERNAME: __token__
```

Comment:
so, we're still using twine, we're just no longer accessing the credentials here, right? and instead we're using the `gh-action-pypi-publish`

Reply:
Yes, I think internally the package does probably use twine, but we don't have to worry about it any more.